### PR TITLE
Eobs 753 high clinical risk process change

### PR DESCRIPTION
**Pull request by Gimpneek**
- Adding test_util methods
- Adding ability to have triggered tasks on cancelling notification

### CODE REVIEW

File: nh_eobs_mobile/static/src/js/nhlib.js [hunk 1]
```diff
@@ -828,23 +828,40 @@ NHMobileForm = (function(superClass) {
...
-          ref1 = self.form.elements;
+          var k, len2, ref2, ref3, results;
+          ref2 = self.form.elements;
           results = [];
-          for (j = 0, len1 = ref1.length; j < len1; j++) {
-            element = ref1[j];
-            if ((ref2 = element.getAttribute('type')) === 'submit' || ref2 === 'reset') {
+          for (k = 0, len2 = ref2.length; k < len2; k++) {
+            element = ref2[k];
+            if ((ref3 = element.getAttribute('type')) === 'submit' || ref3 === 'reset') {
```
Comment (moderatemisbehaviour) [inline nh_eobs_mobile/static/src/js/nhlib.js]: Why is `ref3` assigned here? I can't see it used anywhere.

Comment (Gimpneek) [inline nh_eobs_mobile/static/src/js/nhlib.js]: This is compile sauce from this CoffeeScript 
```
action_buttons = (element for element in self.form.elements \
          when element.getAttribute('type') in ['submit', 'reset'])
```
so `ref3` in this case is for the `type` attribute and then is used in the later part of the OR

Comment (moderatemisbehaviour) [inline nh_eobs_mobile/static/src/js/nhlib.js]: Ah okay, didn't realise this was generated code actually, but thanks for the explanation.

**Comment by moderatemisbehaviour** [comment]
:LGTM: :shipit:

**Comment by Nayira** [comment]
Test for EOBS-753 passed :shipit:

**Pull request merged**